utils/swift-xcodegen/Sources/Xcodeproj/PropertyList.swift

```diff
@@ -104,6 +87,17 @@ fileprivate extension PropertyList {
         self += "   "
       }
     }
+
+    /// Appends the given string, with instances of quote (") and backward slash
+    /// (\) characters escaped with a backslash.
+    mutating func appendEscaped(_ string: String) {
+      for char in string.utf8 {
+        if char == UInt8(ascii: "\\") || char == UInt8(ascii: "\"") {
+          append(UInt8(ascii: "\\"))
```

Comment:
```suggestion
          append(UInt8(ascii: #"\"#))
```
🙂

Reply:
When there's only a single character to escape I tend to prefer just writing the backslash, especially as things like `#"""#` look weird. I think I'd rather keep things locally consistent here, I don't feel too strongly about it though

Reply:
I see. I do not actually have a preference, but I was curious about yours, on the off chance that a raw string had occurred to you.